code/modules/projectiles/magazines/flamer.dm

```diff
@@ -257,10 +257,9 @@
 	matter = list("metal" = 50) // no free metal
 	flamer_chem = null
 	max_rounds = 200
-	max_range = 7
+	max_range = 6
 	fuel_pressure = 1
 	max_duration = 50
```

Comment:
```suggestion
	max_duration = 50
	max_intensity = 50
```

Reply:
They are too powerful, not plentiful (at least haven't seen 3 with it at the same time). It replaces Pyro with research which is no ok. I'm not ok even with napalmx level fire for non specialist (replaces pyro without custom tank), but flamer_tank/custom was added 3 years ago, and wasn't such big of a deal.
What do you mean by "have them scale faster"?

Reply:
See the parameters for https://github.com/cmss13-devs/cmss13/blob/31e1ffbacbed8194287286d1092e16456592539b/code/datums/research_upgrade_datum.dm#L115-L135

It is currently 300 biomass to produce one, requires the first clearance level, and each print of one increases the cost by 100 biomass, to a max of 1000 biomass.

As far as outclassing pyro, OT/researchers can also give pyro better chems in a /obj/item/ammo_magazine/flamer_tank/custom/large which doesn't need any unlock or has any scaling cost.

Reply:
I'd like to note that the clearance level is mostly irrelevant as you need clearance 5 anyways to produce something capable of reaching the tank's caps

Reply:
If you have chems you can use tank infinitely (scaling and initial value doesnt matter). Also, 300 biomass is nothing for brand new pyro specialist.